src/System.Private.Xml/src/System/Xml/XPath/Internal/XPathArrayIterator.cs

```diff
@@ -85,6 +85,6 @@ public override IEnumerator GetEnumerator()
             return list.GetEnumerator();
         }
 
-        private object debuggerDisplayProxy { get { return index < 1 ? null : (object)new XPathNavigator.DebuggerDisplayProxy(Current); } }
+//        private object debuggerDisplayProxy { get { return index < 1 ? null : (object)new XPathNavigator.DebuggerDisplayProxy(Current); } }
```

Comment:
This is being used, by the DebuggerDisplay attribute.  Please be very careful about deleting things like this.  We should really have tests that validate such attributes, as we do in other projects.